### ISSUE TITLE
Datetime fields to Timestamp fields

### ISSUE THREAD

**Issue by boryn** [bpuig/laravel-subby#43]
IMHO all 'DATETIME' fields should be changed to 'TIMESTAMP' (in the `plan_subscriptions` and `plan_subscription_usage`). 

Using 'TIMESTAMP' type fields we are sure that they are always stored in the same timezone (in UTC), according to docs:
_MySQL converts TIMESTAMP values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. (This does not occur for other types such as DATETIME.)_ 

MySQL just saves "now moment in time" as the timestamp and this "now" is universal independent of which timezone the user was at. Good for comparisons, and not making any error with timezone shifts.

**Comment by bpuig** [comment]
Weird thing happens to me if i put $table->timestamp() type in migrations:
```

 Doctrine\DBAL\Exception 

 Unknown column type "timestamp" requested. Any Doctrine type that you use has to be registered with \Doctrine\DBAL\Types\Type::addType(). You can get a list of all the known types with \Doctrine\DBAL\Types\Type::getTypesMap(). If this error occurs during database introspection then you might have forgotten to register all database types for a Doctrine Type. Use AbstractPlatform#registerDoctrineTypeMapping() or have your custom types implement Type#getMappedDatabaseTypes(). If the type name is empty you might have a problem with the cache or forgot some mapping information.
```
I see dateTime ends up making a timestamp column in database. Can you check if the same happens. See deac6e5

**Comment by boryn** [comment]
Hi! I have not had this issue - tested on Windows with XAMPP and 10.4.11-MariaDB and Debian with mysqld 8.0.24. 
And used `->timestamp` before as well on other machines.

What happens if you explicitly add `->default(null)`?

There is an old library for timestamp when it was not yet supported by dbal: https://github.com/marktopper/doctrine-dbal-timestamp-type - you can give it a try.

**Comment by boryn** [comment]
PS. I just noticed that in the scheduler you use `->timestamp` - it does work there, doesn't it?

```php
$table->timestamp('scheduled_at')->nullable();
$table->timestamp('failed_at')->nullable();
$table->timestamp('succeeded_at')->nullable();
```

**Comment by bpuig** [comment]
Let me try again, maybe it's some weird temp thing

**Comment by bpuig** [comment]
Found the reason, I was not using clear migrations, I was using the ones for migrating v3 to v4. The issue was with `->change()`.

Now I set the changes as statements and everything works. bec9b694a1c3fe3090e09ee6e6b334d2f66316cf
